src/index.html

```diff
@@ -7,8 +7,27 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav>
+        <div class="content">
+          <div class="logo"><a href="#"><img src="../src/images/logo.png" alt="MoyoLogo"></a></div>
+          <ul class="nav__list">
+            <li class="nav__link is-active"><a href="#">apple</a></li>
+            <li class="nav__link"><a href="#">samsung</a></li>
+            <li class="nav__link"><a href="#">smartphones</a></li>
+            <li class="nav__link" data-qa="hover"><a href="#">laptops & computers</a></li>
```

Comment:
`data-qa` attribute should be in the `a` tag. Btw I would like you to add a class to those links.
Change the `li` class name to i.e. nav__item and link itself i.e. nav__link